lib/model.py

```diff
@@ -131,6 +135,14 @@ def __repr__(self) -> str:
         return self.__str__()
 
 
+try:
+    from extra_game_handlers import is_supported_extra  # type: ignore
+except ImportError:
+    def is_supported_extra(challenge: Challenge) -> bool:
+        """Accept every challnege by default."""
```

Comment:
There is a typo here. Also, I would prefer a comment more similar to `game_specific_options` like `"""Return True by default not to add any additional restrictions to challenges."""`, because someone may think that their bot will accept every challenge.